Sources/SecureXPC/Server/XPCAnonymousServer.swift

```diff
@@ -10,16 +10,14 @@ import Foundation
 internal class XPCAnonymousServer: XPCServer {
     private let anonymousListenerConnection: xpc_connection_t
     
-    /// Determines if an incoming request can be handled based on the provided client requirements
-    private let messageAcceptor: SecureMessageAcceptor
-
-    internal init(clientRequirements: [SecRequirement]) {
-        self.anonymousListenerConnection = xpc_connection_create(nil, nil)
-        self.messageAcceptor = SecureMessageAcceptor(requirements: clientRequirements)
+    private let _messageAcceptor: MessageAcceptor
+    override internal var messageAcceptor: MessageAcceptor {
+        _messageAcceptor
```

Comment:
What's the purpose of having one private and one internal var?

Reply:
In the case of `XPCMachServer` I need to make use of the fact that the message accept is actually of type `SecureMessageAcceptor` so I could access its `requirements` property.

For the anonymous and XPC Services servers, I don't have such a need. But I though there still needed to be a backing variable? How would you write this code? I'm guessing I lack some knowledge of how Swift properties work.

Reply:
I'm going to merge in this code, but absolutely can revise this part later once I understand how to better write this code.